atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbReadTable.java

```diff
@@ -62,16 +64,30 @@ public DbReadTable(ConnectionSupplier conns, DbQueryFactory queryFactory) {
 
     public ClosableIterator<AgnosticLightResultRow> getAllRows(
             Iterable<byte[]> rows, ColumnSelection columns, long ts, boolean includeValues) {
+        return getAllRows(rows, columns, ts, includeValues, null);
+    }
+
+    public ClosableIterator<AgnosticLightResultRow> getAllRows(
+            Iterable<byte[]> rows, ColumnSelection columns, long ts, boolean includeValues, Boolean reverseRows) {
```

Comment:
taking a `Boolean` arg here feels weird, why not make this a primitive or enum (ASCENDING, DESCENDING, UNDEFINED) for clarity as caller should expressly know desired order?